contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -13,6 +13,12 @@ import { Lib_SafeMathWrapper } from "../../libraries/wrappers/Lib_SafeMathWrappe
 
 /**
  * @title OVM_ECDSAContractAccount
+ * @dev The ECDSA Contract Account exists for backwards compatibility with Ethereum's Layer 1.
+ * It enables the owner of a Layer 1 account to prove ownership and send transaction from that address on Layer 2. 
```

Comment:
A nit would be that the `ovmCREATEEOA` opcode is what enables sending transactions *from that address*, and this contract makes it so that the transaction format is backwards compatible with ETH1.

Could also be worth noting here that the alternative `eth_sign` scheme is also supported by this contract.